ios/FluentUI.Demo/FluentUI.Demo/Demos/BottomCommandingDemoController.swift

```diff
@@ -175,6 +177,20 @@ class BottomCommandingDemoController: DemoController {
         }
     }
 
+    @objc private func toggleHeroCommandHidden(_ sender: BooleanCell) {
+        for (index, heroItem) in heroItems.enumerated() {
+            if index % 2 == 1 {
```

Comment:
nit: kind of abstract logic that will be hidden every other one? the function name doesn't say it. the label of the toggle doesn't say it.

Reply:
I agree that abstraction does create for a bit unknown & confusion on first sight. I removed the use of the modulo operator across the file and replaced it with the `modifiedCommandIndices` array. I also updated the selector function names to make it clear that calling them would only act on items inside the `modifiedCommandIndices`.